chainercv/datasets/__init__.py

```diff
@@ -1,3 +1,7 @@
+from chainercv.datasets.ade20k.ade20k_semantic_segmentation_dataset import ADE20KSemanticSegmentationDataset  # NOQA
+from chainercv.datasets.ade20k.ade20k_test_image_dataset import ADE20KTestImageDataset  # NOQA
+from chainercv.datasets.ade20k.ade20k_utils import ade20k_label_colors  # NOQA
+from chainercv.datasets.ade20k.ade20k_utils import ade20k_label_names  # NOQA
```

Comment:
According to our naming convention, `ade20k_label_colors` should be `ade20k_semantic_segmentation_label_colors`.

Reply:
https://github.com/chainer/chainercv/blob/master/chainercv/datasets/cityscapes/cityscapes_utils.py#L53

You think this is a problem also?
`label_names` should be changed as well because it can be different from the one used by Instance Segmentation.

Reply:
Yes, that is also a problem. Thank  you for pointing out.

Reply:
To summarize:

+ `ade_label_colors` --> `ade_semantic_segmentation_label_colors`
+ `ade_label_names` --> `ade_semantic_segmentation_label_names`
+ `cityscapes_label_colors` --> `cityscapes_semantic_segmentation_label_colors`
+ `cityscapes_label_names` --> `cityscapes_semantic_segmentation_label_names`

Objects for CamVid need not be changed because this dataset only contains semantic segmentation data.

Reply:
Thank you for summarizing.

 > Objects for CamVid need not be changed because this dataset only contains semantic segmentation data.

I agree with you.